src/extensions/mountainview/MVSortingView/ViewContainerTabBar.tsx

```diff
@@ -12,10 +13,11 @@ type Props = {
 }
 
 const ViewContainerTabBar: FunctionComponent<Props> = ({ views, currentView, onCurrentViewChanged, onViewClosed, active }) => {
-    const handleChange = useCallback((event: React.ChangeEvent<{}>, newTabIndex: number) => {
-        const v = views[newTabIndex]
-        if (v) onCurrentViewChanged(v)
-    }, [onCurrentViewChanged, views])
+    // const handleChange = useCallback((event: React.ChangeEvent<{}>, newTabIndex: number) => {
+    //     console.log('---- handleChange', newTabIndex)
+    //     const v = views[newTabIndex]
+    //     if (v) onCurrentViewChanged(v)
+    // }, [onCurrentViewChanged, views])
```

Comment:
How has this logic changed? It seems like you're now handling changes more proactively instead of having a single `handleChange` function, but I'm not seeing the bigger picture here.

Reply:
for some reason, after some styling changes, the onChange wasn't being fired. After a bit of struggling to figure out why, I decided to just handle the clicks directly. So it was just that I don't have control of the MUI Tabs mechanism and couldn't figure out why it wasn't working.

Reply:
Gotcha, okay. That makes sense. As long as it's working now!